SMLHelper/Patchers/DevConsolePatcher.cs

```diff
@@ -1,58 +0,0 @@
-﻿namespace SMLHelper.V2.Patchers
-{
-    using HarmonyLib;
-    using System;
-    using System.Collections.Generic;
-    using System.Reflection;
-
-    internal class DevConsolePatcher
-    {
-        public static List<CommandInfo> commands = new List<CommandInfo>();
```

Comment:
For anyone wondering - nowhere in SMLHelper were any `CommandInfo`s added to this dictionary, and as this was an internal, unused API, I feel it is safe to remove it.